tests/checkpoint/test_checkpoint_id_pk.py

```diff
@@ -81,13 +81,17 @@ def expected_sql_query_output() -> str:
 
 @pytest.fixture
 def expect_column_values_to_be_in_set() -> gxe.ExpectColumnValuesToBeInSet:
-    return gxe.ExpectColumnValuesToBeInSet(column="animals", value_set=["cat", "fish", "dog"])
+    return gxe.ExpectColumnValuesToBeInSet(
+        column="animals",
+        value_set=["cat", "fish", "dog"],  # type: ignore[arg-type] # FIXME
+    )
```

Comment:
Seeing a similiar issue with the `ValueSet` as we have with `Mostly`

```python
ValueSet = Annotated[_ValueSet, Union[list, set]]
```

See 👇 for more details
https://github.com/great-expectations/great_expectations/pull/10441/files#r1782964972